src/App.tsx

```diff
@@ -10,7 +10,7 @@ function App() {
     <Provider store={store}>
       <TranslationProvider>
         <ThemeProviderWrapper>
-          <NavigationBar />
```

Comment:
Where's routing?

How would you control the redirection itself?

Why nothing is happening when I login with `eve.holt@reqres.in:cityslicka`